tests/test_routes.py

```diff
@@ -141,15 +141,15 @@ def test_get_sub_criteria_theme_answers_field_id(request, client):
     assert response.json[0]["field_id"] == "ieRCkI"
 
 
-def test_update_ar_status_to_completed(request, client):
+def test_update_ar_status(request, client):
```

Comment:
Should we test with multiple statuses?
Can we test that the status has actually updated on the application?

Reply:
now testing with both COMPLETED and QA_COMPLETE statuses, and checking db rows are updated 👍🏾